DXRModules/DeusEx/Classes/DXREvents.uc

```diff
@@ -77,10 +92,20 @@ function SetWatchFlags() {
     case "09_NYC_SHIPBELOW":
         WatchFlag('ShipPowerCut');// sparks of electricity come off that thing like lightning!
         break;
+    case "10_PARIS_CATACOMBS":
+        foreach AllActors(class'JunkieFemale', jf) {
+            if(jf.BindName == "aimee")
+                jf.bImportant = true;
+        }
+        break;
     case "10_PARIS_METRO":
         WatchFlag('M10EnteredBakery');
         WatchFlag('AlleyCopSeesPlayer_Played');
         WatchFlag('assassinapartment');
+
+        foreach AllActors(class'GuntherHermann', gunther) {
+            gunther.bInvincible = false;
```

Comment:
actually idk if this should be in DXREvents or DXRMissions or DXRFixup lol, but it's simple anyways